test/commands/org/login/login.sfdx-url.nut.ts

```diff
@@ -56,4 +56,12 @@ describe('org:login:sfdx-url NUTs', () => {
     const output = getString(result, 'shellOutput.stdout');
     expect(output).to.include(`Successfully authorized ${username} with org ID`);
   });
+
+  it('should authorize an org using sfdx-url (human readable)', () => {
+    env.setString('TESTKIT_EXECUTABLE_PATH', `echo '${authUrl}' | ${env.getString('TESTKIT_EXECUTABLE_PATH')}`);
+    const command = 'org:login:sfdx-url -d --sfdx-url-stdin';
+    const result = execCmd(command, { ensureExitCode: 0 });
+    const output = getString(result, 'shellOutput.stdout');
+    expect(output).to.include(`Successfully authorized ${username} with org ID`);
+  });
```

Comment:
@mshanemc the execCmd function does not have a way to prepend the command with another one. I hacked it using `TESTKIT_EXECUTABLE_PATH` but I'm not sure if it works because I did not understand how to run `nut` tests. Could you run it?